lib/cloudshaper/cli.rb

```diff
@@ -0,0 +1,94 @@
+require 'thor'
+require 'cloudshaper'
+
+module Cloudshaper
+  class CLI < Thor
+    class_option "template_dir", type: "string", default: "templates"
```

Comment:
yeah you're right, we're just going to point them at the root module by name, which will be in its own folder.